friends/src/App.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import FriendsList from './component/FriendsList';
+import Form from './component/Form';
+import axios from 'axios';
+import './App.css';
+
```

Comment:
Great job setting isUpdating into state as well as implementing axios get, put, post and delete!